docs/tutorials/getting-started.md

```diff
@@ -0,0 +1,48 @@
+# Getting started
+
+In this tutorial, we'll walk you through the process of deploying the SAML Integrator charm.
+
+## Requirements
+
+You will need:
+
+* A laptop or desktop running Ubuntu (or you can use a VM).
+* [Juju and Microk8s](https://juju.is/docs/olm/microk8s) installed.
```

Comment:
Interesting, so I guess this integrator will only work for `k8s` charms?

Reply:
Not necessarily, but the tutorial is with k8s. See https://github.com/canonical/saml-integrator-operator#saml-integrator-operator

Reply:
Do you think it is worth noting that it also works on machines? If I read the getting started and it is only for k8s I might incorrectly assume that machines are not supported

Reply:
Added clarification on the top paragraph. Let me know if that seems clearer